neonvm/controllers/virtualmachine_controller.go

```diff
@@ -194,7 +190,7 @@ func (r *VirtualMachineReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 }
 
 // finalizeVirtualMachine will perform the required operations before delete the CR.
-func (r *VirtualMachineReconciler) doFinalizerOperationsForVirtualMachine(ctx context.Context, virtualmachine *vmv1.VirtualMachine) error {
```

Comment:
why do we ignore all the errors here?

Reply:
I don't know, honestly. AFAICT, this _probably_ means that it's possible to leak overlay IP addresses due to transient errors.
cc @cicdteam ?